src/main/java/com/qaprosoft/carina/demo/api/fakestoreapi/cart/GetCartBySortMethod.java

```diff
@@ -0,0 +1,19 @@
+package com.qaprosoft.carina.demo.api.fakestoreapi.cart;
+
+import com.qaprosoft.carina.core.foundation.api.AbstractApiMethodV2;
+import com.qaprosoft.carina.core.foundation.api.annotation.Endpoint;
+import com.qaprosoft.carina.core.foundation.api.annotation.ResponseTemplatePath;
+import com.qaprosoft.carina.core.foundation.api.annotation.SuccessfulHttpStatus;
+import com.qaprosoft.carina.core.foundation.api.http.HttpMethodType;
+import com.qaprosoft.carina.core.foundation.api.http.HttpResponseStatusType;
+import com.zebrunner.carina.utils.R;
+
+
+@Endpoint(url = "${config.env.api_url_t}/carts?sort=${desc}", methodType = HttpMethodType.GET)
```

Comment:
Why you've hardcoded here sorting type? Lets rename desc to sort_type and replace it with arc/desc from test data. Also you can make it as a parameter